docs/running-nio/in-the-cloud.md

```diff
@@ -1,5 +1,59 @@
-# nio in the Cloud
+# nio-managed Cloud Instance
+
+While the true power of nio is experienced when you have multiple [local installations](/running-nio/locally) communicating across a distributed network, we also offer the ability to spin up cloud instances through the **System Designer**.
+
+A nio-managed Cloud Instance allows you to get up and running quickly, and to start building services right away (although our local installation process usually only takes 5 minutes!).
+
+>**[info] Prerequisites**
+>
+>* **a nio account**: [Sign up here](https://app.n.io/signup)
+
+---
+## Create A System
```

Comment:
The rest of our headings are sentence case, so "Create system"

Reply:
I did not notice that.